lib/Core/Traits/PagerfantaFindServiceSearchAdaptersTrait.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Netgen\EzPlatformSiteApi\Core\Traits;
+
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use Netgen\EzPlatformSiteApi\Core\Site\Pagination\Pagerfanta\ContentSearchAdapter;
+use Netgen\EzPlatformSiteApi\Core\Site\Pagination\Pagerfanta\ContentSearchHitAdapter;
+use Netgen\EzPlatformSiteApi\Core\Site\Pagination\Pagerfanta\LocationSearchAdapter;
+use Netgen\EzPlatformSiteApi\Core\Site\Pagination\Pagerfanta\LocationSearchHitAdapter;
+use Pagerfanta\Adapter\AdapterInterface;
+use Pagerfanta\Pagerfanta;
+use Symfony\Component\HttpFoundation\Request;
+
+trait PagerfantaFindServiceSearchAdaptersTrait
```

Comment:
`PagerfantaTrait` ? :)

Reply:
In future maybe we would like to have trait that is internally using FilterService

Reply:
Then just `PagerfantaFindTrait`?

Reply:
Like it, +1, @pspanja wdyt ?

Reply:
👍